lib/backend-api/src/types.rs

```diff
@@ -591,6 +591,12 @@ mod queries {
         pub token: String,
     }
 
+    #[derive(cynic::Enum, Clone, Copy, Debug)]
+    pub enum LogStream {
+        Stdout,
+        Stderr,
```

Comment:
We are missing `Runtime` here, right? (or what did we call it exactly?)

Reply:
Maybe that was not added in the backend?

Reply:
According to the schema returned by backend, `Runtime` is not included:
```
enum LogStream {
  STDOUT
  STDERR
}
```